doc/developer/new_contributor_faq.rst

```diff
@@ -37,6 +37,15 @@ That said, a few places to check for ideas on where to get started:
 
 .. _Algorithms discussion: https://github.com/networkx/networkx/discussions/categories/algorithms
 
+Q: I've found an issue I'm interested in, can I have it assigned to me?
+-----------------------------------------------------------------------
+
+NetworkX doesn't typically assign issues to contributors. If you find an issue
+or feature request on the issue tracker that you'd like to work on, you should
+first check the issue thread to see if there are any linked pull requests.
+If not, then feel free to open a new PR to address the issue - there's no need
```

Comment:
Nit: Maybe mention that they may want to reference the issue # when they make the PR.

Reply:
Added an extra note about it in 15ecf8e